### PR TITLE
bug: handle "missing" quality

### PR DESCRIPTION
**Pull request by lasrinnil**
## References

* #886

### CODE REVIEW

File: source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using FluentAssertions;
+using Xunit;
+using QuantityQuality = Energinet.DataHub.Wholesale.Domain.ProcessStepResultAggregate.QuantityQuality;
+
+namespace Energinet.DataHub.Wholesale.WebApi.UnitTests.Infrastructure.Processes;
+
+public class QuantityQualityMapperTests
+{
+    [Fact]
+    public void MapQuality_whenCalled_MapsCorrectly()
```
Comment (BjarkeMeier) [inline source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs:25]: nit
```suggestion
    public void MapQuality_ReturnsMappedQuality()
```

File: source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        // Arrange
+        foreach (var type in Enum.GetValues(typeof(QuantityQuality)))
```
Comment (BjarkeMeier) [inline source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs:28]: Have you considered simplifying the test itself by using xUnits [memberdata](https://hamidmosalla.com/2017/02/25/xunit-theory-working-with-inlinedata-memberdata-classdata/) feature?

**Review by BjarkeMeier — APPROVED** [review]

File: source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs (hunk 2 above)
Comment (lasrinnil) [reply source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs:28]: No I have not, perhaps you would like to introduce it to the whole team at another time :)

**Review by BjarkeMeier — APPROVED** [review]

File: source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs (hunk 2 above)
Comment (BjarkeMeier) [reply source/dotnet/Services/WebApi.UnitTests/Infrastructure/Processes/QuantityQualityMapperTests.cs:28]: Sure thing.

**Pull request merged**